React/Modules/RCTClipboard.m

```diff
@@ -23,15 +23,19 @@ - (dispatch_queue_t)methodQueue
 
 RCT_EXPORT_METHOD(setString:(NSString *)content)
 {
+#if !TARGET_OS_TV
```

Comment:
It would be better to just exclude the whole module. That way you can feature-check it from JS (if `NativeModules.Clipboard`